packages/developer-console-spa/src/components/ConfigureLighthouse.tsx

```diff
@@ -0,0 +1,237 @@
+                  <EmptyStateSecondaryActions>
+                    <Button variant="link" onClick={ openOPLighthouseDoc }>Learn more</Button>
+                  </EmptyStateSecondaryActions>
+                </EmptyState>
+              </CardBody>
+            </Card>
+          </StackItem>
+        :
+          <StackItem>
+            <Card>
+              <CardHeader>
+                <CardActions>
+                  <Button
+                    aria-label="Delete Branch"
+                    alt="Delete Branch" variant="link"
+                  >
+                    <TrashIcon />
+                  </Button>
+                </CardActions>
+                <CardTitle>
+                  <CheckCircleIcon /> You are all set
+                </CardTitle>
+              </CardHeader>
+              <CardBody>
+                <Flex>
+                  <FlexItem align={ { default: 'alignRight' } }>
+                    <Button variant="link" onClick={ openLighthouseDoc }>Learn More</Button>
+                  </FlexItem>
+                  <FlexItem>
+                    <Button variant="link">View my Lighthouse reports    <ExternalLinkAltIcon /></Button>
+                  </FlexItem>
+                </Flex>
+              </CardBody>
+            </Card>
+            <br />
+            <Card id="card2" isExpanded={ isCardExpanded }>
+              <CardHeader
+                onExpand={ onCardExpand }
+                toggleButtonProps={ {
+                  id: 'toggle-button',
+                  'aria-label': 'Details',
+                  'aria-labelledby': 'titleId toggle-button',
+                  'aria-expanded': isCardExpanded
+                } }
+              >
+                <CardTitle>
+                  How do you use lighthouse CI?
+                </CardTitle>
+              </CardHeader>
+              <CardExpandableContent>
+                <CardBody>
+                  <p>To get started using Lighthouse, just add the following code into 'lighthouserc.js' file to your project root:</p>
+                  <ClipboardCopy isExpanded={ true } isCode isReadOnly hoverTip="Copy" clickTip="Copied" variant={ ClipboardCopyVariant.expansion }>
+                    {
+                      `module.exports = {
+                      ci: {
+                          collect: {
+                              url: '${ window.location.origin + app.path }'
+                          },
+                          assert: {
+                              preset: 'lighthouse:recommended'
+                          },
+                      },
+                    };`
+                    }
+                  </ClipboardCopy>
+                  <br />
+                  <p>And you can run the lighthouse tests for your app on your dev environment, or in your CI scripts using:</p>
+                  <ClipboardCopy isCode isReadOnly hoverTip="Copy" clickTip="Copied">
+                    { `lhci autorun -upload.token <your_build_token>` }
```

Comment:
As its a simple string can we convert it back to simple html one rather than a templating one

Reply:
Done